app/assets/scripts/vendor/map.js

```diff
@@ -74,4 +74,35 @@
         setupClickListener('changetype-all', []);
         setupClickListener('changetype-address', ['address']);
         setupClickListener('changetype-establishment', ['establishment']);
+
+        $.ajax({
+          type: "GET",
+          url: "/get-resource"
+        }).done(function(data){
+            data = JSON.parse(data)
+            for(var i = 0; i < data.length; i++){
+            //we have the pins data, access it here and place pins on map
```

Comment:
Nit: capitalize comment and end with a period. `// We have the pins data. Access it here and place pins on map.`